common/changes/@uifabric/merge-styles/const-enum_2018-11-14-00-40.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/merge-styles",
+      "comment": "DevExp: get rid of const enum so that tsloader works in transpileOnly mode",
```

Comment:
I think the changefile should just directly referenced `isolatedModules` being the primary motivator and not tsloader.